app/models/au_user.rb

```diff
@@ -10,7 +10,9 @@ def self.create_with_omniauth(auth)
       au_user.uid = auth["uid"]
       au_user.name = auth["info"]["name"]
       au_user.email = auth["info"]["email"]
-      if au_user.avatar
+      if auth["info"]["pictures"].nil?
```

Comment:
You don't need to change this, but I'm wondering:
* Why is this using an empty string instead of leaving it as a not required field & therefore OK to be `nil` field?